ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py

```diff
@@ -281,7 +288,7 @@ def _get_l2_as_subint(self, link: dict, svi: dict, vrf: dict) -> dict:
 
         # TODO: enable flow tracking once toggle is in place
```

Comment:
you can remove the TODO's in your PR

Reply:
Resolving this being tracked by comment 
https://github.com/aristanetworks/avd/pull/3776/files#diff-2450b7720f928002e1393655d5f09465e22042c4a868b2621667aa2989124e03